analytics/views/stats.py

```diff
@@ -49,23 +49,35 @@ def is_analytics_ready(realm: Realm) -> bool:
 def render_stats(
     request: HttpRequest,
     data_url_suffix: str,
-    target_name: str,
+    realm: Optional[Realm],
+    *,
+    label: Optional[str] = None,
```

Comment:
I was wondering if there's a better name for this than `label`? We use it for the browser tab/title and the page header in `stats.html`.

Reply:
Yes, `title` feels the best to me.